src/libYARP_math/include/yarp/math/FrameTransform.h

```diff
@@ -13,121 +13,70 @@
 #include <yarp/math/Math.h>
 #include <yarp/math/Quaternion.h>
 
+#define VALID_POSITION         1  // 0b00000001
+#define VALID_ROTATION         2  // 0b00000010
+#define VALID_LIN_VELOCITY     4  // 0b00000100
+#define VALID_LIN_ACCELERATION 8  // 0b00001000
+#define VALID_ANG_VELOCITY     16 // 0b00010000
+#define VALID_ANG_ACCELERATION 32 // 0b00100000
+#define VALID_FULL VALID_POSITION | VALID_ROTATION | VALID_LIN_VELOCITY | VALID_LIN_ACCELERATION | VALID_ANG_VELOCITY | VALID_ANG_ACCELERATION
+
```

Comment:
Are those deines intended to be used by the user application or only for internal usage?
In the latter case, we shall move them in the `.cpp`, in the former can they be enum or at least can the name be more specific, like `TF_VALID_FULL`? I'm a bit worried about possible name conflict with other interfaces or similar.